hack/update-codegen.sh

```diff
@@ -18,25 +18,29 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SCRIPT_PACKAGE=k8s.io/cluster-registry
-SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
+SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 SCRIPT_BASE=$(cd ${SCRIPT_ROOT}/../..; pwd)
+SCRIPT_PACKAGE=$(basename $(dirname "${SCRIPT_ROOT}"))
```

Comment:
This only works if the script is only one level below the real source root right? Why not just encode that here instead of this indirect encoding?

Reply:
I'm not quite sure I follow what you're asking me to do. What would be the direct version of encoding this?

Reply:
By direct encoding I meant specifying that you are asking for the parent's basename by writing something like `basename $(readlink -f "${SCRIPT_ROOT}/..")`. But this is fine.